src/userdoc/web-ui/browse/comment.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Commenting documents
+review:
+    comment: ''
+    date: '2018-11-14'
+    status: ok
+description: null
+toc: true
+labels:
+    - comments
+tree_item_index: 300
+
+---
+{{! excerpt}}
+Users with read access can comment to express their point of view on the document. To have better collaboration on content, users can create, edit, and delete threaded comments on documents.
+{{! /excerpt}}
+
+## Creating a comment
+
+In the Comments section of the Document view, type your comment and click Enter.
+
+![]({{file name='comment-writing.png'}} ?w=300,border=true)
+
+## Replying to a comment
+
+To reply to a comment, click ![]({{file name='reply.png'}} ?w=20) next to the comment you want to reply to.
+
+Type your comment and click Enter.
+
+![]({{file name='reply-comment.png'}} ?w=300,border=true)
+
+## Editing a comment or a reply
```

Comment:
Indicate if you can edit the comment of somebody else (permissions). 
Indicate you can only have a 2 level thread (not infinite)